src/main/java/com/EighthLight/app/Game.java

```diff
@@ -0,0 +1,42 @@
+package com.EighthLight.app;
+
+public class Game implements com.EighthLight.app.IGame {
+    private IPlayer playerOne;
+    private IPlayer playerTwo;
+    private IUserInterface ui;
+    private Board board;
+
+    public Game(IUserInterface ui, Board board, IPlayer playerOne, IPlayer playerTwo) {
+        this.ui = ui;
+        this.board = board;
+        this.playerOne = playerOne;
+        this.playerTwo = playerTwo;
+    }
+
+    public void start() {
+        boolean gameIsTie = false;
+        ui.display(ui.presentBoard(board.getSpaces()));
```

Comment:
you're calling two UI methods here. one as an argument to another. When I see this, I question why presenting the board doesn't just call display after it's determined how to present itself. small nitpick.